Sources/Honeycomb/Honeycomb.swift

```diff
@@ -96,8 +96,11 @@ public class Honeycomb {
             }
         let spanProcessor = BatchSpanProcessor(spanExporter: spanExporter)
 
+        let baggageSpanProcessor = HoneycombBaggageSpanProcessor(filter: { _ in true })
+
         let tracerProvider = TracerProviderBuilder()
             .add(spanProcessor: spanProcessor)
+            .add(spanProcessor: baggageSpanProcessor)
```

Comment:
is this part of the URLSession instrumentation?

Reply:
Sort of. It's something we do in our other SDKs, and by adding it here, I can more easily to implement the smoke tests, by setting baggage before I make the request, and then checking that those ended up on the spans.

I could pull it out into its own PR, but since it's so small and should be uncontroversial, I was just gonna do it here. But if you feel strongly, I can pull it out.